javascript/javascript2/week1/movieApp/index.js

```diff
@@ -1,259 +1,141 @@
+  if (minutes >= 59) {
+    hours++;
+    minutes = 0;
   }
-
-  const stars = ratingsDiv.querySelectorAll("i");
-
-  for (let star of stars) {
-    star.addEventListener("click", () => {
-      const children = star.parentElement.children;
-      for (let child of children) {
-        if (child.getAttribute("data-clicked")) {
-          return false;
-        }
-      }
-
-      star.setAttribute("data-clicked", "true");
-      const rating = star.dataset.rating;
-      movie.rating = rating;
-      /* set rating  */
-      console.log(movie);
-    });
+  if (hours >= 24) {
+    hours = 0;
   }
-
-  detailsDiv.appendChild(ratingsDiv);
-
-  return cardContainer;
 }
 
-/* search section */
-const searchDiv = document.createElement("div");
-searchDiv.setAttribute("class", "searchDiv");
-const searchBox = document.createElement("input");
-searchBox.placeholder = "Search For Movie";
-searchBox.setAttribute("class", "searchMovie");
-const searchButton = document.createElement("button");
-searchButton.innerText = "Search";
-searchBox.setAttribute("class", "searchInput");
-searchButton.setAttribute("class", "searchButton");
-header.appendChild(searchDiv);
-searchDiv.appendChild(searchBox);
-searchDiv.appendChild(searchButton);
-
-searchButton.addEventListener("click", () => {
-  const searchedInput = searchBox.value.toLocaleLowerCase();
-  const searchedMovie = getMoviesbyNameSearch(searchedInput);
-  displayMovies(searchedMovie);
-});
-
-/* nav Ul */
-const headerUl = document.createElement("ul");
-headerUl.setAttribute("class", "headerUl");
-navLinkList.forEach((item) => {
-  const li = document.createElement("li");
-  li.innerText = item;
-  headerUl.appendChild(li);
-});
-
-header.appendChild(headerUl);
-mainApp.appendChild(header);
-/* section for sort by category  */
-const categoryList = document.createElement("div");
-categoryList.setAttribute("class", "stickyCategory");
-mainApp.appendChild(categoryList);
-
-const categoryUl = document.createElement("ul");
-categoryUl.setAttribute("class", "categoryUl");
-categoryList.appendChild(categoryUl);
-
-let filteredMovie = [];
-
-movieCategory.forEach((category) => {
-  const li = document.createElement("li");
-  li.innerText = category;
-  categoryUl.appendChild(li);
-  li.addEventListener("click", () => {
-    const allCategories = categoryUl.querySelectorAll("li");
-    allCategories.forEach((i) => i.classList.remove("active"));
-    li.classList.add("active");
-    if (category === "All") {
-      displayMovies(movieList);
-    } else {
-      filteredMovie = getMoviesbyCategory(category);
-      displayMovies(filteredMovie);
-    }
-  });
-});
-/* functions for filtering */
-function getMoviesbyNameSearch(title) {
-  return movieList.filter((movie) => movie.title.toLowerCase() === title);
-}
-function getMoviesbyCategory(category) {
-  return movieList.filter((movie) => movie.genre === category);
+function renderTime(item) {
+  const secsInFormat = seconds < 10 ? `0${seconds}` : seconds;
```

Comment:
You can let the browser do the formatting with [padStart](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart).
```suggestion
  const secsInFormat = seconds.toString().padStart(2, '0');
```